packages/core/src/lib/InputNew/atoms/VisibilityToggle.tsx

```diff
@@ -12,6 +12,7 @@ const VisibilityToggle: React.FC<IVisibilityToggleProps> = ({
         className="input_visibility"
         data-testid="test-input-icon-visibility"
         onClick={() => onClick()}
+        tabIndex={-1}
```

Comment:
why minus -1?

Reply:
on password type input, when you click tab it focuses on the password icon to avoid this negative index is being used. (Never seen any site implement focusing on the password icon)

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex
```
A negative value (usually tabindex="-1") means that the element is not reachable via sequential keyboard navigation, but could be focused with JavaScript or visually by clicking with the mouse. It's mostly useful to create accessible widgets with JavaScript.
```